suite-common/formatters/src/tests/cryptoToFiatAmountFormatter.test.ts

```diff
@@ -0,0 +1,348 @@
+import { prepareCryptoToFiatAmountFormatter } from '../formatters/prepareCryptoToFiatAmountFormatter';
+
+const coins = [
+    {
+        current: {
+            ts: 1674639055000,
+            symbol: 'eth',
+            rates: {
+                yfi: 0.2093598,
```

Comment:
nit: do you need all the currencies here? Isn't just usd/eur/czk enough for tests for example?